Assets/Scripts/UI/PickupPup/DogProfile/DogShelterProfile.cs

```diff
@@ -24,6 +25,7 @@ protected override void setReferences()
     {
         base.setReferences();
         iconsObject.SetActive(false);
+        adoptUIButton = adoptButton.GetComponent<UIButton>();
```

Comment:
UIButton does not provide any external functionality to adjust its image component. Its selectedColor and deselectedColor are hardcoded. Are you asking me to change that?